src/components/@common/Dropdown/index.tsx

```diff
@@ -0,0 +1,134 @@
+import { forwardRef } from 'react';
+import {
+  ClearIndicatorProps,
+  components,
+  DropdownIndicatorProps,
+  MultiValueRemoveProps,
+} from 'react-select';
+import ArrowDown from '../SVG/Icon/ArrowDown';
+import Close from '../SVG/Icon/Close';
+import * as Styled from './Dropdown.styled';
+
+/**
+ *  @Dropdown
+ *    @사용목적
+ *      1) 다양한 화면에서 재사용 가능한 Dropdown UI 제공
+ *    @주요기능
+ *      1) 단일 선택 및 다중 선택 지원
+ *      2) react-hook-form과의 통합 지원
+ */
+
+export interface IDropdownProps {
+  /**
+   * 전체 Dropdown 컴포넌트(Label + DropDown 포함)의 가로 길이를 설정
+   * CSS 단위(px, %, rem 등)를 포함하여 전달
+   * @default "320px"
+   */
+  width?: string;
+
+  /** Dropdown 상단에 표시할 텍스트 */
+  label?: string;
+
+  /**
+   * Dropdown에 표시할 옵션 배열
+   * 각 옵션은 value와 label 속성을 포함하는 객체로 구성
+   * - value: 옵션의 실제 값
+   * - label: 표시될 텍스트
+   * @example
+   * options={[
+   *    {value: 1, label: 'Option 1'}
+   *    {value: 2, label: 'Option 2'}
+   * ]}
+   */
+  options: {
+    value: string | number;
+    label: string;
+  }[];
+
+  placeholder?: string;
+
+  /**
+   * 해당 Dropdown이 필수 입력값인지 여부 설정
+   * @default false
+   */
+  isRequired?: boolean;
+
+  /**
+   * 다중 선택 모드 활성화 여부 설정
+   * @default false
+   */
+  isMulti?: boolean;
+
+  /** react-hook-form의 Controller로부터 전달된 현재 값 */
+  value?: { value: number | string; label: string };
+
+  /** react-hook-form의 Controller로부터 전달된 onChange Handler */
+  onChange?: (value: any) => void;
+}
+
+const DropdownIndicator = (props: DropdownIndicatorProps) => {
+  return (
+    <components.DropdownIndicator {...props}>
+      <ArrowDown title="arrowDown" width="24px" height="24px" />
```

Comment:
```suggestion
      <ArrowDown title="드롭다운 메뉴 열기" width="24px" height="24px" />
```
svg 태그의 title 속성은 스크린 리더가 svg 내용을 설명할 때 사용하는걸로 알고있는데, 
해당 부분을 위와 같이 좀 더 명시적으로 써주면 웹 접근성 측면에서 좀 더 좋을 것 같습니다.

Reply:
꼼꼼한 리뷰 감사드립니다! 🙇

말씀해주신 내용을 반영하여, 웹 접근성 측면에서 더 적합하도록 모든 SVG의 `title`을 수정하였습니다.
추가로 보완할 점이 있으면 언제든 말씀해주세요! 😄